improver_tests/generate_ancillaries/test_GenerateClearskySolarRadiation.py

```diff
@@ -163,6 +174,118 @@ def test__irradiance_times():
         )
 
 
+def test__calc_air_mass():
```

Comment:
Please make use of `pytest.mark.parametrize` to remove all the duplication in your tests. The other tests would benefit from this too.

```suggestion
@pytest.mark.parametrize(
    "zenith, expected_values", 
    (
        (-100.0, 0.0), 
        (-90, 0.0), 
        (-85, 11.46028), 
        etc...
    )
)
def test__calc_air_mass(zenith, expected_values):
```